backend/eventloop/application_event_loop/application_event_runner_test.go

```diff
@@ -1130,7 +1130,7 @@ var _ = Describe("ApplicationEventLoop Test", func() {
 				},
 				Type: sharedutil.ManagedEnvironmentSecretType,
 				Data: map[string][]byte{
-					"kubeconfig": ([]byte)(kubeConfigContents),
+					shared_resource_loop.Kubeconfig_key: ([]byte)(kubeConfigContents),
```

Comment:
Shouldn't the variables be in camel case ? like `KubeconfigKey` and all small case for the package name `sharedresourceloop` ?

Reply:
Thanks Anand, I'll correct it to `KubeconfigKey` although the package name is `shared_resource_loop` and it is being used in multiple places throughout the code